### PR TITLE
[MPQEditor] Implement providing model nodes info.

### PR DESCRIPTION
**Pull request by stamalakhov**
This commit implements providing model nodes info.

Fresh draft: https://github.com/Samsung/ONE-vscode/pull/1511
Previous draft: https://github.com/Samsung/ONE-vscode/pull/1505
Related: https://github.com/Samsung/ONE-vscode/issues/1491

ONE-vscode-DCO-1.0-Signed-off-by: s.malakhov <s.malakhov@partner.samsung.com>

### CODE REVIEW

File: src/MPQEditor/MPQEditor.ts [hunk 1]
```diff
@@ -267,6 +269,91 @@ export class MPQEditorProvider implements vscode.CustomTextEditorProvider {
...
+    const K_EXIT: string = "exit";
+    const K_ERROR: string = "error";
+    let modelFilePath = this.getModelFilePath(document);
+
+    // TODO integrate with Toolchain
+    const tool = "/usr/share/one/bin/circle-operator";
+    if (!fs.existsSync(tool)) {
+      // check whether it is installed
+      Balloon.info(
+        "To add more layers for editing please install ONE-toolchain"
```
Comment (seanshpark) [inline src/MPQEditor/MPQEditor.ts:299]: @jyoungyun , is `ONE-toolchain` name appropriate? or if not, can you please suggest?

Comment (stamalakhov) [reply src/MPQEditor/MPQEditor.ts:299]: > @stamalakhov How about using just `Toolchain`? ONE-vscode supports two types of toolchains, ONE and TRIV. And each toolchain contains `one-compiler` so user can use `onecc` command in any toolchain.

Ok. I see. I'll change it to `Toolchain`. Thank you.

Comment (stamalakhov) [reply src/MPQEditor/MPQEditor.ts]: @seanshpark @jyoungyun 
done.

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by jyoungyun — APPROVED** [review]
LGTM

**Review by dayo09 — APPROVED** [review]
LGTM, sorry for the late response.

**Pull request merged**